### DISCUSSION TITLE
[help] Pipeline seems to load full dynamic target into memory when it is used as a dependency

### DISCUSSION THREAD

**Discussion by kendonB** [ropensci/targets#1146] [Help]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

No reprex yet as it's tricky to demo memory stuff and you may know what's going on.

I have a targets with code:

```
    simulations_w_metadata = tar_target({
        grps |>
            slice(1) |>
            mutate(data = list(simulations))
    }, pattern = map(grps, simulations)),
```
One item of simulations is small, but in aggregate they're large. `grps` is a small df with length N and `simulations` is a dynamic target with length N. Any ideas for troubleshooting?

**Comment by kendonB** [comment]
Hmm even attempting to run this one as a slurm job with worker loading and storing blows up memory. Perhaps the parallel `pattern = map()` has an issue?

**Comment by kendonB** [comment]
Oops, I am mixing the `target_name = command` and `tar_target` syntax. likely causing the issue. If there's any change warranted it's probably to make the `tar_plan` code error when seeing something like this.

**Reply by wlandau** [reply]
Glad you figured it out
